src/controllers/csi/driver/volumes/app/publisher.go

```diff
@@ -87,6 +97,11 @@ func (publisher *AppVolumePublisher) UnpublishVolume(ctx context.Context, volume
 	}
 	log.Info("loaded volume info", "id", volume.VolumeID, "pod name", volume.PodName, "version", volume.Version, "dynakube", volume.TenantUUID)
 
+	if volume.MountAttempts > 0 {
```

Comment:
i don't quite understand that:
* the publisher tries to mount a volume for `bindCfg.MaxMountAttempts` times
* only if `volume.MountAttempts > bindCfg.MaxMountAttempts` a dummy volume is mounted
* so wouldn't it be technically possible for a real, non-dummy volume to be correctly mounted even if it has `> 0` mount attempts, as long as it is `<= bindCfg.MaxMountAttempts`?

Reply:
if a Volume was mounted correctly we set the MountAttempts to `0`

https://github.com/Dynatrace/dynatrace-operator/pull/1109/files#diff-8f7ea714d1cb32f86ee9d8d9b5e11671a9f52de76970b12bb57dc203308ea537R83
https://github.com/Dynatrace/dynatrace-operator/pull/1109/files#diff-8f7ea714d1cb32f86ee9d8d9b5e11671a9f52de76970b12bb57dc203308ea537R226
https://github.com/Dynatrace/dynatrace-operator/pull/1109/files#diff-8f7ea714d1cb32f86ee9d8d9b5e11671a9f52de76970b12bb57dc203308ea537R244